trie/zk_trie.go

```diff
@@ -106,9 +114,13 @@ func (t *ZkTrie) UpdateContractCode(_ common.Address, _ common.Hash, _ []byte) e
 //
 // The value bytes must not be modified by the caller while they are
 // stored in the trie.
-func (t *ZkTrie) Update(key, value []byte) {
-	if err := t.TryUpdate(key, value); err != nil {
-		log.Error(fmt.Sprintf("Unhandled trie error: %v", err))
+func (t *ZkTrie) Update(key, value []byte) error {
+	return t.TryUpdate(key, value)
+}
+
+func (t *ZkTrie) MustUpdate(k, v []byte) {
+	if err := t.TryUpdate(k, v); err != nil {
+		panic(err)
```

Comment:
Unlike other must functions, handling panic is a bit awkward. Why not make it consistent?

Reply:
Oh, I didn't think much about it. Most of them don't cause panic, they take error logs. I'll change it